nautobot_chatops/dispatchers/base.py

```diff
@@ -41,11 +41,14 @@ def subclasses(cls):
         if settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_slack"):
             from .slack import SlackDispatcher
 
-        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("ms_teams"):
+        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_ms_teams"):
             from .ms_teams import MSTeamsDispatcher
 
-        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("webex_teams"):
-            from .webex_teams import WebExTeamsDispatcher
+        if settings.PLUGINS_CONFIG["nautobot_chatops"].get("enable_webex"):
+            from .webex import WebExDispatcher
```

Comment:
Do we also need a `enable_webex_teams` section here?

Reply:
That's a good point I didn't catch.  Updated